src/modules/tests/test_data/test-data.actions.ts

```diff
@@ -22,6 +22,7 @@ export const TOGGLE_PLANNING_ECO = '[Eco] Toggle Planning Eco';
 export const TOGGLE_CONTROLLED_STOP = '[ControlledStop] Toggle Controlled Stop';
 export const CONTROLLED_STOP_COMPLETE = '[ControlledStop] Controlled Stop Complete';
 export const CONTROLLED_STOP_PENDING = '[ControlledStop] Controlled Stop Pending';
+export const ADD_DRIVING_FAULT_COMMENT = '[Competency] Add driving fault comment';
```

Comment:
Does the `Competency` component dispatch this action?

Reply:
Done